client/my-sites/comments/comment-list/index.jsx

```diff
@@ -37,22 +40,37 @@ export class CommentList extends Component {
 		} );
 	}
 
-	toggleCommentLike = commentId => {
-		const comments = map( this.state.comments, comment => {
-			if ( commentId === comment.ID ) {
-				const newLike = ! comment.i_like;
-				// If like changes to true, also approve the comment
+	getEmptyMessage = () => {
+		const { status, translate } = this.props;
+
+		switch ( status ) {
+			case 'unapproved':
 				return {
-					...comment,
-					i_like: newLike,
-					status: newLike ? 'approved' : comment.status,
+					title: translate( 'No new comments yet.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'No new comments yet!' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/comments/placeholders)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).